dev/data/globalrole.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "kind": "GlobalRole",
```

Comment:
I'm wondering if we should hardcode this role and created it during the initialization of the database

Reply:
For now, it filled with provisioning. I didn't have the need to hardcode it maybe later?

Reply:
I think it would depend if without this role, we can use Perses.
If the answer is no, then we need to hardcode these roles to have then in the database.

What you are using here is a development environment. You need to think what would happen if you start the binary with no pre-configuration or initial database

Reply:
we discussed offline with @Gladorme and at the end we agreed that if we deactivate by default de permissions it will simplify the default config and the init of the database for Perses.